test-case/check-audio-equalizer.sh

```diff
@@ -26,16 +26,16 @@ OPT_PARM_lst['d']=1         OPT_VALUE_lst['d']=5
 OPT_OPT_lst['l']='loop'     OPT_DESC_lst['l']='loop count'
 OPT_PARM_lst['l']=1         OPT_VALUE_lst['l']=1
 
-OPT_OPT_lst['s']='sof-logger'   OPT_DESC_lst['s']="Open sof-logger trace the data will store at $LOG_ROOT"
+OPT_OPT_lst['s']='sof-logger'   OPT_DESC_lst['s']="Open sof-logger trace the data will store =at $LOG_ROOT"
 OPT_PARM_lst['s']=0             OPT_VALUE_lst['s']=1
 
-func_opt_parse_option $*
+func_opt_parse_option "$*"
```

Comment:
The code in and around this functions seems to be a mess of `$*, "$*", $@,...` but please keep it a _consistent_ mess for now so if something fails it fails the same all across the board :-(

https://mywiki.wooledge.org/Quotes#Expand_argument_lists
https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html